modules/ossm-federation-config-smcp.adoc

```diff
@@ -120,15 +120,15 @@ To avoid naming conflicts between meshes, you must create separate egress and in
 |spec:
   gateways:
     additionalEgress:
-      <egressName>:
+      <egress_name>:
```

Comment:
If we want to adopt a convention for using snake_case for anything contained within angle brackets, should [everything on this page](https://73013--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-federation) be changed to follow this convention?

Reply:
@eoinfennessy yes, that's the goal per the official style guide.

Reply:
created OSSM-5966 to capture the work on other pages. using this PR only for changes on this PR as this was technically scope creep that delayed the change associated with the intent of the initial story.  :)